src/utils.ts

```diff
@@ -10,36 +12,34 @@ export function getDayOfMonth(date: number): number {
   return date % 100;
 }
 
-export function calcEndTime(startTime: number, duration: number) {
-  const hours = Math.floor(duration);
-  const minutes = (duration % 1) * 100;
-
-  let newHours = Math.floor(startTime / 100) + hours;
-  let newMinutes = (startTime % 100) + minutes;
-
-  // carry the 1 from minutes
-  if (newMinutes >= 60) {
-    newMinutes -= 60;
-    newHours += 1;
-  }
-
-  return newHours * 100 + newMinutes;
-}
-
 export function getDateTime(date: number, time: number): string {
   const day = date % 100;
   const month = Math.floor(date / 100) % 100;
   const year = Math.floor(date / 10000);
 
-  const hours = Math.floor(time / 100);
-  const minutes = time % 100;
+  const hours = Math.floor(time);
```

Comment:
fixed how we calculate time

Reply:
I'm a bit confused how this might work.
I'm assuming the time is given as an integer so 10:30 is 1030. If we just do Math.floor, won't it just make hours = 1030?